Service.Core/Kernel/Kernel.Core/ArgParser.hpp

```diff
@@ -7,8 +7,8 @@
 * @class ArgParser.
 * @brief This class parses arguments with which application is started.
 * @details
-* For example: "--serverport=65001", "--port=5432", "--dbname=juniorgram", "--hostaddr=127.0.0.1", "--user=postgres", "--password=postgres".
-* Sending the port to the Server class constructor and list of all arguments to the PostgreAdapter Instance Method.
+* For example: "--kernelport=65001", "--port=5432", "--dbname=juniorgram", "--hostaddr=127.0.0.1", "--user=postgres", "--password=postgres".
```

Comment:
Right...
I don't like kernel here at all. Let it be `port_service` or `serviceport`. Find out better option

Reply:
Done